public/views/content/addStudent.html

```diff
@@ -33,7 +33,7 @@
       </div>
 
       <div class="form-group">
-          <button class="btn btn-default" ng-click = "addStudent()" type="submit">Submit</button>
+          <button class="btn btn-default" ng-disabled="!newStudent.archived" ng-click = "addStudent()" type="submit">Submit</button>
```

Comment:
I guess it could have been require that OR the other somehow? 

This made more sense to me.